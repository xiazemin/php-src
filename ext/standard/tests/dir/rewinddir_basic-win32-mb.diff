012+   string(51) "私はガラスを食べられますfile1.tmp"
012-   string(45) "私はガラスを食べられますfile1.tmp"
024+   string(51) "私はガラスを食べられますfile2.tmp"
024-   string(45) "私はガラスを食べられますfile2.tmp"