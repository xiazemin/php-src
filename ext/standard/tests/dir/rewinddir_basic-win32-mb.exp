*** Testing rewinddir() : basic functionality ***
resource(%d) of type (stream)
resource(%d) of type (stream)

-- Read and rewind first directory (argument supplied) --
array(3) {
  [0]=>
  string(1) "."
  [1]=>
  string(2) ".."
  [2]=>
  string(45) "私はガラスを食べられますfile1.tmp"
}
NULL
bool(true)

-- Read and rewind second directory (no argument supplied) --
array(3) {
  [0]=>
  string(1) "."
  [1]=>
  string(2) ".."
  [2]=>
  string(45) "私はガラスを食べられますfile2.tmp"
}
NULL
bool(true)
===DONE===