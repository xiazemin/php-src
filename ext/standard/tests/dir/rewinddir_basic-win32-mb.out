*** Testing rewinddir() : basic functionality ***
resource(10) of type (stream)
resource(11) of type (stream)

-- Read and rewind first directory (argument supplied) --
array(3) {
  [0]=>
  string(1) "."
  [1]=>
  string(2) ".."
  [2]=>
  string(51) "私はガラスを食べられますfile1.tmp"
}
NULL
bool(true)

-- Read and rewind second directory (no argument supplied) --
array(3) {
  [0]=>
  string(1) "."
  [1]=>
  string(2) ".."
  [2]=>
  string(51) "私はガラスを食べられますfile2.tmp"
}
NULL
bool(true)
===DONE===